modules/wp-seo/wp-seo-front.php

```diff
@@ -4,7 +4,19 @@
 }
 
 function qtranxf_wpseo_add_filters_front() {
+    // Use indexation only in backend on "publish/update" events to save all languages data in indexable tables.
+    // If you allow indexation on the frontend - then indexable table will save data of the first visited page, all other languages data will be missed/filtered/gone/didn't saved.
+    // After all this fixes everyone need to install "Yoast Test Helper", go to "Instruments -> Yoast Test" and reset "Indexables & migrations table", then go to "SEO -> Instruments" and start SEO data optimization.
+    // This filters "wpseo_should_save_indexable", "wpseo_indexing_data" - right now works only on frontend, so using of "!is_admin()" is pointless, but maybe in future this filters will work on backend, that's why I have leaved "!is_admin()" below.
+    // Unfortunately I didn't found how to totally disable indexable feature, Yoast documentation is very poor to quickly understand how everything works.
+    // Also be advised that after Yoast have added indexables - now you also need to provide breadcrumb titles and its translations even if they are same as in title. If you wouldn't provide breadcrumb title & translation - Yoast will only index current language title.
+    if ( ! is_admin() ) {
```

Comment:
If it's admin it should rather go to `wp-seo-admin.php`.

Reply:
Oh, sorry, didn't mention that the filename called "wp-seo-FRONT" :)
In that case you can delete condition `if ( ! is_admin() )`

Reply:
Done.